internal/models/api/common.go

```diff
@@ -0,0 +1,10 @@
+package api
```

Comment:
Let's add package level comments,

one line comments for each `type`s, and for each field within the type.

Reply:
Ok
Should I add for older parts as well?

Reply:
We can gradually add for older parts as we touch them.

Reply:
I've added some documentation
I just found this blog post that defines how to do go documentation - https://go.dev/blog/godoc
Seems like there is no docstring constructs and we just need to write comments on top of types, functions etc. No conventions given either.

Please take a look

Reply:
[This](https://elliotchance.medium.com/godoc-tips-tricks-cda6571549b) might help understand conventions @aneeshsharma

Reply:
Thanks @maaverik 
I need to add full stops to paragraphs
I think right now they will be recognized as headings